.circleci/config.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     docker:
       # specify the version you desire here
       # use `-browsers` prefix for selenium tests, e.g. `3.6.1-browsers`
-      - image: circleci/python:3.6.1
+      - image: circleci/python:3.7.1
```

Comment:
Do we want to change this to 3.8.1? or 3.9.0?

Reply:
I guess we need to test several of the currently actively used ones